scania-digital-design-system/sdds
# fix(icon): migration

**Describe pull-request**  
Migrating Icon component

**Solving issue**  
Fixes: [AB#2397](https://dev.azure.com/scaniadigitaldesignsystem/3d9eb9c6-0045-473d-bc1d-e842dd779200/_workitems/edit/2397)

**How to test**  
1. Go to Netlify link below
2. Check Foundation -> Icon
3. Play with both component and font version to see if it works OK
